src/modules/Common/containers/Layout.tsx

```diff
@@ -38,7 +38,7 @@ const Layout = ({
           <Header>
             {() => (
               <>
-                <HeaderMenu>{t('header:title')}</HeaderMenu>
+                <HeaderMenu></HeaderMenu>
```

Comment:
why a empty header menu ? maybe put `&nbsp;` at least as it seems weird

Reply:
I deleted this empty component.